pkg/tfgen/generate_python.go

```diff
@@ -711,8 +712,20 @@ func (g *pythonGenerator) emitPackageMetadata(pack *pkg) error {
 	w.Writefmtln("class InstallPluginCommand(install):")
 	w.Writefmtln("    def run(self):")
 	w.Writefmtln("        install.run(self)")
-	w.Writefmtln("        check_call(['pulumi', 'plugin', 'install', 'resource', '%s', '${PLUGIN_VERSION}'])",
+	w.Writefmtln("        try:")
+	w.Writefmtln("            check_call(['pulumi', 'plugin', 'install', 'resource', '%s', '${PLUGIN_VERSION}'])",
 		pack.name)
+	w.Writefmtln("        except OSError as error:")
+	w.Writefmtln("            if error.errno == errno.ENOENT:")
+	w.Writefmtln("                print(\"\"\"")
+	w.Writefmtln("                There was an issue installing the resource provider plugin.")
```

Comment:
s/issue/error/

Reply:
We should probably mention the provider name here too, so: "There was an error installing the X resource provider plugin."

Reply:
This is copied exactly from the NodeJS version...

Reply:
Updated and also changed to pulumi.com rather than pulumi.io